agent-install/agent-install.sh

```diff
@@ -456,20 +456,20 @@ function install_linux(){
     fi
 
 	log_info "Checking if the agent port ${ANAX_PORT} is free..."
-	if [ ! -z "$(netstat -nlp | grep :$ANAX_PORT)" ]; then
+	if [ ! -z "$(netstat -nlp | grep \":$ANAX_PORT\")" ]; then
```

Comment:
I dont understand how this fix works:
$ export ABC=53
$ netstat -nlp | grep :${ABC}
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)
tcp        0      0 127.0.0.53:53           0.0.0.0:*               LISTEN      -
udp        0      0 0.0.0.0:5353            0.0.0.0:*                           -
udp        0      0 127.0.0.53:53           0.0.0.0:*                           -
udp6       0      0 :::5353                 :::*                                -
$ netstat -nlp | grep \":${ABC}\"
(Not all processes could be identified, non-owned process info
 will not be shown, you would have to be root to see it all.)

Reply:
There is supposed to be a space after $ANAX_PORT. It's there now. I don't understand why that happened either